src/Simplify.cpp

```diff
@@ -1746,6 +1783,15 @@ class Simplify : public IRMutator2 {
     Expr visit(const Div *op) override {
         Expr a = mutate(op->a);
         Expr b = mutate(op->b);
+
+        if (no_fp_simplify && op->type.is_float()) {
```

Comment:
Might be simpler in all these cases to put:

if (no_fp_simplify && op->type.is_float()) {
  return IRMutator2::mutate(op);
}

above the lines that mutate a and b